wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

```diff
@@ -17,10 +17,10 @@ The **Code of Conduct for Modding and Mapping** is a set of rules and guidelines
 So you downloaded a map and you've thought of some things you aren't happy with - now's the time to put your thoughts to words and help improve the map. But how can you get the mapper to see your point of view, test your suggestions and give you feedback? After all, no-one likes feeling that their time has been wasted.
 
 -   Before modding, ensure your intentions are good and you are focused on the task at hand. No-one likes to receive a rude or lazy mod.
--   When making suggestions, try to differentiate between what is intersubjectively / objectively bad or wrong, and what you simply dislike on a personal level. **All types of suggestion are fine as long as they are put forward accordingly**.
+-   When making suggestions, try to differentiate between what is objectively bad or wrong, and what you simply dislike on a personal level. **All types of suggestion are fine as long as they are put forward accordingly**.
     -   **Objective Issues** - Prohibited by rulesets and must be fixed.
-    -   **Intersubjective Issues** - Not prohibited by rulesets, but are agreed upon as being "wrong" by a majority of people. Should be fixed in most cases unless there are opposing intersubjective arguments with equal size, or if the mapper can exhaustively explain why they feel it is acceptable
-    -   **Subjective Issues** - Something that you believe or think on a personal level. Might become intersubjective if many people agree with it.
+    -   **Social Issues** - Not prohibited by rulesets, but are agreed upon as being "wrong" by a majority of people, usually because they impinge upon or challenge established traditions or "unspoken" expectations as a whole. These should be fixed in most cases unless there is considerable, well-founded reasoning to the contrary, as many of these expectations have been naturally created across many years of collaborative creation.
```

Comment:
For me, the way this first is worded now swings a little too far in the other direction - that merely attempting to do something different is wrong.

I think you can strike a better balance by somehow clarifying that it's challenging the expected norms in a _disruptive_ way that we're trying to dissuade, rather than trying to change anything at all, under any circumstance.

Reply:
it's also missing the "unless there are opposing intersubjective arguments with equal size" part, I think that's still relevant to include because not every established tradition will be consistent across all of osu!'s mappers/modders